script/installer.js

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
+
+const createWindowsInstaller = require('electron-winstaller').createWindowsInstaller
+const path = require('path')
+const rimraf = require('rimraf')
+
+deleteOutputFolder()
+  .then(getInstallerConfig)
+  .then(createWindowsInstaller)
+  .catch((error) => {
+    console.error(error.message || error)
+    process.exit(1)
+  })
+
+function getInstallerConfig() {
+  const rootPath = path.join(__dirname, '..')
+  const outPath = path.join(rootPath, 'release')
+
+  return Promise.resolve({
+    appDirectory: path.join(outPath, 'Mattermost-win32-ia32'),
```

Comment:
It seems that electron-winstaller can handle x64 app because of the sample written on README.md. I'm using x64 Windows, so I will check later.